src/templates/kit/screens/TokensScreen.js

```diff
@@ -52,28 +52,25 @@ function validationError(tokenName, tokenSymbol, members) {
 }
 
 function Tokens({
-  back,
-  data,
-  fields,
-  next,
-  screenIndex,
-  screens,
   accountStake = -1,
+  dataKey = 'tokens',
+  screenProps: { back, data, next, screenIndex, screens },
 }) {
+  const screenData = (dataKey ? data[dataKey] : data) || {}
```

Comment:
No need for this no? `dataKey` has a default value

Reply:
That’s because `dataKey` can be set to `null` explicitly, in which case the keys will be added do `data` directly.